### PR TITLE
wip: Release documentation

### PR DESCRIPTION
**Pull request by Dreznel**
Need to include:
* List of supported features
* Descriptions of any odd behavior (such as logging config)
* Descriptions of commands and flags.

### CODE REVIEW

File: MANUAL.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+  ```
+
+## Default Settings
+Below are the default settings for non-required pkgr configurations.
+  * Debug mode: disabled
+  * Preview mode: disabled
+  * Strict mode: disabled
+  * Loglevel: “info”
+  * Rpath: “R”
+  * Threads: Number of processors available on system
```
Comment (dpastoor) [inline MANUAL.md]: threads not processors - eg on my 6 core mac --> will default to 12 threads.

File: MANUAL.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  * Cache: System temp directories
```
Comment (dpastoor) [inline MANUAL.md]: User cache directory (not system) - under the hood this uses ~ os.UserCacheDir() which is I believe like ~/.cache/... on unix systems

Comment (Dreznel) [reply MANUAL.md]: > this is looking really good/helpful. The one thing is I think this needs to either be in the wiki _or_ the readme. I doubt people will find the manual (I'm not aware of other repos that have a manual.md), so if we're going to go off the readme, we might as well use a "real" system - aka the wiki, that can be maintained/extended as large as we need it to be.

I agree that the Wiki is the place to put this ultimately, I'll transfer the information over there once we've gotten everything straight with the testing docs, as those are currently cluttering it up.

**Comment by Dreznel** [comment]
> this is looking really good/helpful. The one thing is I think this needs to either be in the wiki _or_ the readme. I doubt people will find the manual (I'm not aware of other repos that have a manual.md), so if we're going to go off the readme, we might as well use a "real" system - aka the wiki, that can be maintained/extended as large as we need it to be.

I agree that the Wiki is the place to put this ultimately, I'll transfer the information over there once we've gotten everything straight with the testing docs, as those are currently cluttering it up.

**Comment by Dreznel** [comment]
I have made the requested changes and ported the MANUAL over to [the wiki](https://github.com/metrumresearchgroup/pkgr/wiki/user_manual).

**Pull request closed without merging**